### ISSUE TITLE
About the loss 

### ISSUE THREAD

**Issue by Rain-Y** [shelldream/DSSM#1]
delta_sim = pos_sim - neg_sim
Then ,I think we should make the delta_sim max

self.loss = tf.reduce_mean(tf.sigmoid(delta_sim))
self.optimizer = optimizer.minimize(self.loss)

But, minimize the loss seemly makes the delta_sim min.
